src/twisted/words/protocols/irc.py

```diff
@@ -69,6 +70,9 @@
 CHANNEL_PREFIXES = "&#!+"
 
 
+MSG_OR_NOTICE = enum.Enum("MSG_OR_NOTICE", "PRIVMSG NOTICE")
```

Comment:
As this enum isn't used by any public API surface it should have a private name.

I think it'd be fine &mdash; even preferable &mdash; to pass the strings `'PRIVMSG'` and `'NOTICE'` directly without the enum at all, again because it's all private. Enums are slow after all.

Reply:
I prefer passing enums over string constants as it's easier to spot typos that way, but I guess that's not a prblem here as the usage is very limited.